mlops/ProjectFile.py

```diff
@@ -3,15 +3,23 @@
 """
 import yaml
 import datetime
+import os
 
 
-class ProjectFile:
+class ProjectFile():
 
-    def __init__(self, config, projectfile_name='MLProject'):
+    def __init__(self, config, path='.', projectfile_name='MLProject', use_localhost=False):
         self.config = config
         self.projectfile_name = projectfile_name
+        self.project_path = os.path.join(path, projectfile_name)
+
+        if use_localhost:
+            volume_path = config['project']['LOCAL_VOLUME_MOUNT']
+        else:
+            volume_path = config['project']['VOLUME_MOUNT']
+
         self.project_dict = {'name': config['project']['NAME'].lower(),
-                             'docker_env': {'volumes': [config['project']['VOLUME_MOUNT']],
+                             'docker_env': {'volumes': [volume_path],
```

Comment:
why does this need to be a list?

Reply:
Not entirely sure of the reason, but there's an exception to catch it if its not a list!

`mlflow.exceptions.ExecutionException: Project configuration (MLproject file) was invalid: Docker volumes must be a list of strings, e.g.: '["/path1/:/path1", "/path2/:/path2"])`